### PR TITLE
chore: update to use axe-core 4.4.3

### PR DESCRIPTION
**Pull request by michael-siek**
Ref: #239

### CODE REVIEW

File: selenium/package.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "repository": "dequelabs/axe-core-maven-html",
   "scripts": {},
   "dependencies": {
-    "axe-core": "^4.4.2"
+    "axe-core": "^4.4.3"
   },
   "devDependencies": {
-    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures"
+    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures#v1"
```
Comment (Zidious) [inline selenium/package.json:12]:
```suggestion
    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures"
```

File: playwright/package-lock.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
       "version": "4.4.1",
       "license": "MPL",
       "dependencies": {
-        "axe-core": "^4.4.2"
+        "axe-core": "^4.4.3"
```
Comment (Zidious) [inline playwright/package-lock.json:12]: Good catch - I must of missed this in a previous PR, when installing `axe-test-fixtures` as we denoted `^` for axe-core it updated it to 4.4.3 within [node_modules](https://github.com/dequelabs/axe-core-maven-html/pull/240/files#diff-d3b2d086922e59583106c513a7abfb49f34cdaf70e13ec300f4b564f84a26fcdR43-R48)

**Review by Zidious — CHANGES_REQUESTED** [review]
Minor thing with fixture dependency

File: selenium/package.json (hunk 1 above)
Comment (michael-siek) [reply selenium/package.json:12]: So the base branch in this repo is v1 so this should be correct https://github.com/dequelabs/axe-test-fixtures/

**Pull request merged**